ros_ign_gazebo_demos/launch/imu.launch.py

```diff
@@ -15,52 +15,48 @@
 import os
 
 from ament_index_python.packages import get_package_share_directory
+
 from launch import LaunchDescription
 from launch.actions import DeclareLaunchArgument
 from launch.actions import IncludeLaunchDescription
-from launch.conditions import IfCondition
 from launch.launch_description_sources import PythonLaunchDescriptionSource
-from launch.substitutions import LaunchConfiguration
+
 from launch_ros.actions import Node
 
+
 def generate_launch_description():
 
-    pkg_ros_ign_gazebo_demos = get_package_share_directory('ros_ign_gazebo_demos')
```

Comment:
This will be needed when RViz is enabled again, maybe we could leave it here commented out?